inst/components/scss/card.scss

```diff
@@ -1,4 +1,7 @@
 .bslib-card {
+  // TODO: allow a way to opt out
+  overflow: auto;
```

Comment:
Just a note to highlight that fortunately the `class` argument of `card()` is added at this level, so `class = "overflow-visible"` and similar is a pretty natural way to modify this choice.